site/source/_posts/2018/01/17-Announcing-Dojo-2-beta-5/Announcing-Dojo-2-beta-5.md

```diff
@@ -0,0 +1,109 @@
+---
+title: Announcing Dojo 2 beta 5!
+date: 2018-01-17 09:00:00
+authorId: dylan
+featured_image: featured.png
+---
+
+Dojo is a modern, reactive framework authored with ES2017 and TypeScript, and optimized with the needs of enterprise application development in mind.
+
+If you're learning about Dojo 2 for the first time, please read our overview about the [Dojo 2 beta 4 release](https://dojo.io/blog/2017/12/04/Announcing-Dojo-2-beta-4/).
+
+{% asset_img featured.png feature-full %}
+
+<!-- more -->
+
+## Getting Beta 5
+
+Beta 5 is currently tagged as `latest` in npm, so simply updating to the latest version will provide you with the Beta 5 release.  We also have a release tag of `beta5` in npm if you want to focus just on using Beta 5.
+
+This will be the final beta release for Dojo 2! As we make releases of packages in anticipation of Release Candidate 1, we will not publish those packages to `latest` until Release Candidate 1 is available.
+
+## Just getting started
+
+If you want to get started with Dojo 2, the best way is to take a look at our [tutorials](/tutorials/) or even take a look at Dojo 2 in action with our [example applications](https://dojo.github.io/examples/) (source on [GitHub](https://github.com/dojo/examples)).  Good starting tutorials are:
+
+* [Dojo 2 local installation](/tutorials/000_local_installation/)
+* [Your first Dojo 2 application](/tutorials/001_static_content/)
+* [Creating widgets](/tutorials/003_creating_widgets/)
+
+## New in beta 5
+
+Since the beta 4 release, we have made a number of improvements. The biggest changes and additions are mentioned here, with a complete list of changes
+
+### Initial re-architecture of the build system
```

Comment:
Worth including `cli-create-theme`? maybe not in this section because it's about the build system but somewhere?

Reply:
Added